app/api/auth/[...nextauth]/route.ts

```diff
@@ -41,17 +41,18 @@ const handler = NextAuth({
     }),
   ],
   callbacks: {
-    async jwt({ token, user }) {
+    async jwt({ token, user }: { token: any; user: any }) {
       return { ...token, ...user };
     },
-    async session({ session, token }) {
+    async session({ session, token }: { session: any; token: any }) {
```

Comment:
Can we replace the `any` type here? (At least for the token)

Reply:
I managed to do that for the token, however replacing the other types crashes the application and I can't seem to find the correct type in next-auth docs